backend/manager/modules/utils/src/main/java/org/ovirt/engine/core/utils/ovf/OvfReader.java

```diff
@@ -518,7 +518,7 @@ private VmDevice readOtherHardwareItem(XmlNode node) {
                     .forValue(String.valueOf(selectSingleNode(node, VMD_TYPE, _xmlNS).innerText));
             String device = selectSingleNode(node, VMD_DEVICE, _xmlNS).innerText;
             // special devices are treated as managed devices but still have the OTHER OVF ResourceType
-            managed = VmDeviceCommonUtils.isSpecialDevice(device, type, true);
+            managed = VmDeviceCommonUtils.isSpecialDevice(device, type, true) || type == VmDeviceGeneralType.TPM;
```

Comment:
I think it would be nicer to call `readManagedVmDevice(node, readDeviceId(node))` when detecting TPM in readHardwareSection so TPM won't be considered as "other hardware" anywhere

Reply:
OK, done.